mockall/src/lib.rs

```diff
@@ -1605,7 +1605,8 @@ impl Times {
     /// Has this expectation already been called the minimum required number of
```

Comment:
You should update doc comment too, because you've changed the function's behavior.